Tests/XCUITests/SaveLoginsTests.swift

```diff
@@ -90,15 +90,11 @@ class SaveLoginTest: BaseTestCase {
         waitForExistence(app.tables["Login List"])
         XCTAssertTrue(app.staticTexts[domain].exists)
         // XCTAssertTrue(app.staticTexts[domainSecondLogin].exists)
-        if processIsTranslatedStr() == m1Rosetta {
-            XCTAssertEqual(app.tables["Login List"].cells.count, defaultNumRowsLoginsList + 2)
+        // Workaround for Bitrise specific issue. "vagrant" user is used in Bitrise.
+        if (ProcessInfo.processInfo.environment["HOME"]!).contains("vagrant") {
```

Comment:
The same issue on the number of logins saved appear on both Intel and M1 Bitrise stacks. How do we detect if the test is being executed on Bitrise instead of MacStadium or local machines?

Reply:
Is this line not working: `if processIsTranslatedStr() == m1Rosetta` this has been the case so far for these type of tests...

This test has been working fine on both environment (with the issue with the user name not detected, an automation issue) for a long time. 
Why don't we keep that line?

Reply:
We have been using Intel on Bitrise, so testing for `m1Rosetta` worked in the past. :(

Reply:
I tried to query environment variables that are set only in the Bitrise environment but wasn't successful. The list of environment variables (`$BITRISE_*`) are found from Bitrise's workflow editor. 😞

Reply:
@isabelrios What are the other ways to query the environment? 🙏🏼 We have used `ProcessInfo.processInfo` in our tests.